File: java/Pc.java
public class Pc
{
    private Processor processor;
    private Disk disk;
    private Ram ram;
    private int totalPrice = 0;

    Pc(){}

    Pc(Processor processor, Disk disk, Ram ram)
    {
        this.processor = processor;
        this.disk = disk;
        this.ram = ram;
        this.totalPrice = processor.getPrice() + disk.getPrice() + ram.getPrice();
    }

    public void setProcessor(Processor processor)
    {
        this.processor = processor;
    }

    public Processor getProcessor()
    {
        return this.processor;
    }

    public void setDisk(Disk disk)
    {
        this.disk = disk;
    }

    public Disk getDisk()
    {
        return this.disk;
    }

    public void setRam(Ram ram)
    {
        this.ram = ram;
    }

    public Ram getRam()
    {
        return this.ram;
    }

    public void setTotalPrice()
    {
        this.totalPrice = this.processor.getPrice() + this.disk.getPrice() + this.ram.getPrice();
    }

    public int getTotalPrice()
    {
        return this.totalPrice;
    }

    public void showPc()
    {
        this.processor.showProcessor();
        this.disk.showDisk();
        this.ram.showRam();

        this.setTotalPrice();
        System.out.println("Total Price     : " + this.getTotalPrice());
    }
}

File: python/Pc.py
from Processor import Processor
from Disk import Disk
from Ram import Ram

class Pc:
    processor = None
    disk = None
    ram = None
    totalPrice = 0

    def __init__(self, processor, disk, ram):
        self.processor = processor
        self.disk = disk
        self.ram = ram
        self.totalPrice = processor.getPrice() + disk.getPrice() + ram.getPrice()

    def setProcessor(self, processor):
        self.processor = processor
    
    def getProcessor(self):
        return self.processor

    def setDisk(self, disk):
        self.disk = disk

    def getDisk(self):
        return self.disk

    def setRam(self, ram):
        self.ram = ram

    def getRam(self):
        return self.ram

    def setTotalPrice(self):
        self.totalPrice = self.processor.getPrice() + self.disk.getPrice() + self.ram.getPrice()

    def getTotalPrice(self):
        return self.totalPrice

    def showPc(self):
        self.processor.showProcessor()
        self.disk.showDisk()
        self.ram.showRam()
        
        self.setTotalPrice()
        print("Total Price     : " + str(self.getTotalPrice()))